mxd/MXD_dataexchange_tutorial1.md

```diff
@@ -0,0 +1,151 @@
+
+## Provide data
+
+In this step we will focus on inserting data into our participant Alice using
+the [Management API](https://app.swaggerhub.com/apis/eclipse-edc-bot/management-api/0.1.4-SNAPSHOT). We will use plain
+CLI tools (`curl`) for this, but feel free to use graphical tools such as Postman or Insomnia.
+
+Alice, as a data consumer, wants to consume data from Bob. Bob, as a data provider, needs to create an asset for Alice. The data asset should have the following properties:
+
+| ID          | 1                                                                                            |
+|-------------|----------------------------------------------------------------------------------------------|
+| Description | Tractus-X EDC Demo Asset                                                                     |
+| Type        | HttpData                                                                                     |
+| URL         | [https://jsonplaceholder.typicode.com/todos/1](https://jsonplaceholder.typicode.com/todos/1) |
+
+Action (Bob): Create this asset using the following curl command:
+
+```shell
+curl -X POST "${BOB_DATAMGMT_URL}/data/assets" \
+    --header 'X-Api-Key: password' \
+    --header 'Content-Type: application/json' \
+    --data '{
+             "asset": {
+                "properties": {
+                        "asset:prop:id": "1",
+                        "asset:prop:description": "Tractus-X EDC Demo Asset"
+                    }
+                },
+                "dataAddress": {
+                    "properties": {
+                        "type": "HttpData",
+                        "baseUrl": "https://jsonplaceholder.typicode.com/todos/1"
+                    }
+                }
+            }' \
+    -s -o /dev/null -w 'Response Code: %{http_code}\n'
+```
+
+Bob tells Alice, that he created an asset, and she should now be able to request it. In the next step, Alice requests a contract offer catalog. In this catalog, all contract offers for Alice are listed.
+
+Action (Alice): Execute a request using the following curl commands:
+
+```shell
+curl -G -X GET "${ALICE_DATAMGMT_URL}/data/catalog" \
+    --data-urlencode "providerUrl=${BOB_IDS_URL}/api/v1/ids/data" \
+    --header 'X-Api-Key: password' \
+    --header 'Content-Type: application/json' \
+    -s | jq
+```
+
+Let´s see if Alice can see the Asset. Can you find it?
+
+As you can see in the response, the data offer "Tractus-X EDC Demo Asset" does not appear. Unfortunately, Alice sees some contract offers but she cannot find the contract offer from Bob.
+
+Alice calls Bob and says she can´t see the asset. Bob remembers that he did not create an access policy. An access policy defines who is allowed to see a data offering. To create a policy that allows Alice to access the data offering, Bob needs Alice's Business Partner Number (BPN). Alice´s BPN is BPNL000000000001.
```

Comment:
This BPN Number part should be moved to next tutorial. In this tutorial we will just create a unrestricted policy (dummy policy) so that anyone can access the assets using this access policy. We will add restrictions in the next tutorial.
In the below curl for access policy, you can see there are no BPN number.